sheerwater_benchmarking/forecasts/ecmwf_er.py

```diff
@@ -574,19 +114,15 @@ def ifs_extended_range(start_time, end_time, variable, forecast_type,
     # Convert to base180 longitude
     ds = lon_base_change(ds, to_base="base180")
 
-    if variable == 'tmp2m':
+    if variable in ['tmp2m', 'tmax2m', 'tmin2m']:
         ds[variable] = ds[variable] - 273.15
         ds.attrs.update(units='C')
     elif variable == 'precip':
         ds[variable] = ds[variable] * 1000.0
         ds.attrs.update(units='mm')
         ds = np.maximum(ds, 0)
-        if time_group == 'weekly':
```

Comment:
Removed this, but haven't re-run the ecmwf caches. I've kept a divide by 7 in the forecast function. Should update this.